CONTRIBUTING.md

```diff
@@ -0,0 +1,281 @@
+# Contributing
+
+Contributions are welcome and greatly appreciated! There are several ways to
+contribute to XTGeo.
+
+## Types of Contributions
+
+### Report Bugs
+
+Report bugs at
+[https://github.com/equinor/xtgeo/issues](https://github.com/equinor/xtgeo/issues).
+
+If you are reporting a bug, please include:
+
+- Your operating system name and version.
+- Any details about your local setup that might be helpful in troubleshooting.
+- Detailed steps to reproduce the bug.
+
+### Fix Bugs
+
+Look through the GitHub issues for bugs. Anything tagged with "bug", "help
+wanted", or "good first issue" is open to whomever wants to fix it.
+
+### Implement Features
+
+Look through the GitHub issues for features. Anything tagged with
+"enhancement", "help wanted", or "good first issue" is open to whomever wants
+to implement it.
+
+### Write Documentation
+
+XTGeo could always use improved documentation, whether as part of the
+official XTGeo docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+### Submit Feedback
+
+The best way to send feedback is to file an issue at
+[https://github.com/equinor/xtgeo/issues](https://github.com/equinor/xtgeo/issues).
+
+If you are proposing a feature:
+
+- Explain in detail how it would work.
+- Keep the scope as narrow as possible, to make it easier to implement.
+
+## Get Started!
+
+Ready to contribute? Here's how to set up XTGeo for local development.
+
+1. Fork the XTGeo repository from the Equinor repository to your GitHub
+   user account.
+
+2. Clone your fork locally:
+
+   ```sh
+   git clone git@github.com:your_name_here/xtgeo
+   cd xtgeo
+   git remote add upstream git@github.com:equinor/xtgeo
+   git remote -v
+   # origin	git@github.com:your_name_here/xtgeo (fetch)
+   # origin	git@github.com:your_name_here/xtgeo (push)
+   # upstream	git@github.com:equinor/xtgeo (fetch)
+   # upstream	git@github.com:equinor/xtgeo (push)
+   ```
+
+3. Install your forked copy into a local venv:
+
+   ```sh
+   python -m venv ~/venv/xtgeo
+   source ~/venv/xtgeo/bin/activate
+   pip install -U pip
+   pip install -e ".[dev,docs]"  # add -vv to see compilation output
+   ```
+
+4. Install the test data one directory below your XTGeo directory and run
+   the tests to ensure everything works:
+
+   ```sh
+   git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+   pytest -n auto
+   ```
+
+5. Create a branch for local development:
+
+   ```sh
+   git checkout -b name-of-your-bugfix-or-feature
+   ```
+
+   Now you can make your changes locally.
+
+6. When you're done making changes, check that your changes pass ruff and the
+   tests:
+
+   ```sh
+   ruff check .
+   ruff format .
+   mypy src/
+   pytest tests
+   ```
+
+7. If you want to edit the C/C++ contact the XTGeo authors for detailed
+   instructions.
```

Comment:
Propose better wording: `If you want to contribute to the C/C++ code base, contact the ...`

Reply:
Thanks, I mangled that sentence